### PR TITLE
change dir to untar cli in ansible

### PR DESCRIPTION
**Pull request by csantanapr**
cc @dubeejw @houshengbo

### CODE REVIEW

File: ansible/roles/cli/tasks/download_openwhisk_cli.yml [hunk 1]
```diff
@@ -55,13 +55,14 @@
...
     src: "{{ cli.nginxdir }}/{{ os }}/{{ arc }}/{{ dest_name }}"
     dest: "{{ cli.nginxdir }}/{{ os }}/{{ arc }}/"
     remote_src: True
   when: os != "linux"
 
-- name: "unarchive cli ({{ os }}) {{ arc }}"
-  local_action: shell "tar" "-zxvf" "{{ dest_name }}" chdir="{{ cli.nginxdir }}/{{ os }}/{{ arc }}"
+- name: "unarchive cli ({{ os }}) {{ arc }} for linux only"
+  shell: "tar -zxvf {{ dest_name }}"
+    chdir: "{{ cli.nginxdir }}/{{ os }}/{{ arc }}"
```
Comment (dubeejw) [inline ansible/roles/cli/tasks/download_openwhisk_cli.yml]: Maybe need:

```
args:
     chdir: ...
```

Comment (houshengbo) [inline ansible/roles/cli/tasks/download_openwhisk_cli.yml]: @csantanapr @dubeejw I have already sumitted another PR and merged. now waiting for the result.

**Comment by houshengbo** [comment]
@csantanapr @dubeejw I have already sumitted another PR and merged. now waiting for the result.

**Comment by houshengbo** [comment]
@csantanapr @dubeejw My fix in openwhisk does not resolve the issue, it still cannot find the path .

File: ansible/roles/cli/tasks/download_openwhisk_cli.yml [hunk 2]
```diff
@@ -55,13 +55,14 @@
     headers: "{{ headers }}"
     mode: 0755
 
-- name: "unarchive cli ({{ os }}) {{ arc }}"
+- name: "unarchive cli ({{ os }}) {{ arc }} for not linux"
   unarchive:
     src: "{{ cli.nginxdir }}/{{ os }}/{{ arc }}/{{ dest_name }}"
```
Comment (houshengbo) [inline ansible/roles/cli/tasks/download_openwhisk_cli.yml]: This PR will not work either. I have forced Linux to use this line to untar, and it did not work either, still not found the path.

**Comment by markusthoemmes** [comment]
@csantanapr could you add a description on what is failing in this case?

**Comment by csantanapr** [comment]
staled PR

**Pull request closed without merging**